src/Entity/IdeasWorkshop/Vote.php

```diff
@@ -32,25 +50,34 @@ class Vote
     /**
      * @ORM\ManyToOne(targetEntity="Idea", inversedBy="votes")
      * @ORM\JoinColumn(nullable=false)
+     *
+     * @Assert\NotNull
```

Comment:
maybe `NotBlanc`? `NotNull` allows to have a empty string `""`

Reply:
This field in the database is integer, an empty string could not be safe and it's weird to have `NotBlank` on a non string property, no?